src/fontra/client/core/ui-utils.js

```diff
@@ -243,6 +243,7 @@ export const DefaultFormatter = {
   },
 };
 
+// TODO: Maybe move formatters to utils.js, so we can write unittests for it.
```

Comment:
We can consider creating formatters.js, but utils.js is not the right place.

Reply:
I created `formatters.js`, but did not move existing formatters, only the newly added + I added a `TODO` comment. I mark this as resolved.